app/home/const.ts

```diff
@@ -0,0 +1,2 @@
+export const CARD_PREVIEW_IMAGE_SIZE_WIDTH = 300;
+export const CARD_PREVIEW_IMAGE_SIZE_HEIGHT = 300;
```

Comment:
Since you might bring `convertMovieResponseToCard` under `Card` file, you might as well take this one there too as this is a domain knowledge of Card rather than `Home`

Reply:
Relocated the values under util.ts, rather than re-creating const.ts file for now because I wanted to avoid creating many files under a specific directory.

Reply:
Yes that'll do. It feels like the name `util` is a bit too general now. As you are putting functions and variables to be consumed on the client side, would you like to name it something like; `homeClientUtils.ts`?

Reply:
I feel "Client" sounds too dependent on the context, especially in Next.js. The page.tsx also uses the values and the component is considered as server side somponent...

I renamed it as `MovieCard.utils.ts` for now.